server/package.json

```diff
@@ -11,7 +11,6 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "bcrypt": "^5.1.1",
```

Comment:
Why is this removed? I see that you're using it in the `hashPasswordUtils` file.

Reply:
Philip and I were wondering the same thing!  I do not even remember doing that.  I will put it back.